Google.PowerShell.IntegrationTests/Provider/Storage.Provider.Tests.ps1

```diff
@@ -71,6 +71,15 @@ Describe "Storage Provider"{
         Compare-Object $gcsContents $localContents | Should BeNullOrEmpty
     }
 
+    It "Should create file using set-content" {
+        cd gs:\$bucketName\$folderName
+        rm $fileName
+        Set-Content $fileName -Value $content1
+        Sleep 1
```

Comment:
Is this necessary? My understanding is that GCS writes are atomic, thought I wouldn't be surprised if the delay were needed.